src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -301,7 +303,7 @@ public partial class GCExtendedGamepadSnapshot {
 
 		public static bool TryGetSnapShotData (NSData data, out GCExtendedGamepadSnapShotDataV100 snapshotData)
```

Comment:
If the parameter can be null, you should also change it to say as much:

```suggestion
		public static bool TryGetSnapShotData (NSData? data, out GCExtendedGamepadSnapShotDataV100 snapshotData)
```